defaults/main.yml

```diff
@@ -13,8 +13,10 @@ thinlinc_email: "root@localhost"
 thinlinc_printers: "yes"
 
 # Enabled services
-thinlinc_webadmin_enabled: true
+thinlinc_vsmagent_enabled: true
+thinlinc_vsmserver_enabled: true
 thinlinc_webaccess_enabled: true
+thinlinc_webadmin_enabled: true
 
```

Comment:
I don't believe this adds much value to the role. Setting these to false will have no effect on the services. I'd like to keep the default behavior that the services are enabled and started per default.